thirdparty/vars.sh

```diff
@@ -476,9 +476,21 @@ LIBDIVIDE_NAME="libdivide-v5.2.0.tar.gz"
 LIBDIVIDE_SOURCE="libdivide-v5.2.0"
 LIBDIVIDE_MD5SUM="4ba77777192c295d6de2b86d88f3239a"
 
+# pprof
+PPROF_DOWNLOAD="https://github.com/murphyatwork/pprof/releases/download/20251014/pprof-linux-amd64.zip"
```

Comment:
this will break arm platform. either skip the pprof installation on arm or provide arm binary link as well.

Reply:
fixed